### PR TITLE
DSS-2064 adding aws session to client calls

### PR DESCRIPTION
**Pull request by ghasani**
The data apps all crated a AWS session that is authed to aws.

### CODE REVIEW

File: to_data_library/data/transfer.py [hunk 1]
```diff
@@ -294,7 +296,7 @@ def s3_to_gs(self, s3_connection_string, s3_bucket_name,
         logs.client.logger.info(f'Found {str(s3_files)} files in S3')
 
         # For every key found in s3, download to local and then upload to desired GS bucket.
-        s3_client = s3.Client(parsed_connection['region'])
+        s3_client = s3.Client(aws_session, parsed_connection['region'])
```
Comment (Munir93) [inline to_data_library/data/transfer.py:299]: please remove all references to parsed connection as its no longer passed to the client

File: to_data_library/data/transfer.py [hunk 2]
```diff
@@ -328,13 +328,14 @@ def _get_keys_in_s3_bucket(self, parsed_connection, bucket_name, prefix_name):
                 s3_files.append(obj.get('key'))
         return s3_files
 
-    def s3_to_bq(self, s3_connection_string, bucket_name, object_name,
+    def s3_to_bq(self, aws_session, s3_connection_string, bucket_name, object_name,
```
Comment (Munir93) [inline to_data_library/data/transfer.py:331]: Same here, please remove connection string

File: to_data_library/data/transfer.py [hunk 3]
```diff
@@ -368,7 +369,7 @@ def s3_to_bq(self, s3_connection_string, bucket_name, object_name,
         # Download S3 file to local
         parsed_connection = parse.parse(
             '{region}:{access_key}:{secret_key}', s3_connection_string)
-        s3_client = s3.Client(parsed_connection['region'])
+        s3_client = s3.Client(aws_session, parsed_connection['region'])
```
Comment (Munir93) [inline to_data_library/data/transfer.py:372]: And here, if needed please make region a param on the method

**Review by Munir93 — CHANGES_REQUESTED** [review]

File: to_data_library/data/transfer.py [hunk 4]
```diff
@@ -335,7 +335,7 @@ def s3_to_bq(self, aws_session, s3_connection_string, bucket_name, object_name,
         Exports S3 file to BigQuery table
 
         Args:
-          aws_session: authenticated AWS session.
+                aws_session: authenticated AWS session.
           s3_connection_string (str): The S3 connection string in the format
                                       {region}:{access_key}:{secret_key}
```
Comment (kthhrv) [inline to_data_library/data/transfer.py:340]: I don't believe we need to be talking about access_keys and secret_keys anymore? we're just passing around an aws_session

Comment (Munir93) [reply to_data_library/data/transfer.py:340]: Ive requested to remove already @kthhrv :)

**Review by Munir93 — COMMENTED** [review]
All looks good now. Test failing flake 8 linting

**Pull request merged**